src/stylesheets/input_list.scss

```diff
@@ -27,7 +27,6 @@
       display: block;
       background: darken($gray-tint, 5);
       padding: 6px 12px;
-      height: 34px;
```

Comment:
This change is probably safe, but in order to ensure that the element doesn't collapse if it has content that doesn't intrinsically have height of its own, I would suggest changing this property to 
```css
    min-height: 34px;
```
rather than removing height altogether. That will prevent the element collapsing, but will allow it to grow as needed.